aten/src/ATen/native/Unique.cpp

```diff
@@ -0,0 +1,58 @@
+// Returns unique elements of input tensor.
+
+#include "ATen/ATen.h"
+#include "ATen/Dispatch.h"
+
+#include <set>
+#include <tuple>
+#include <unordered_map>
+#include <unordered_set>
+
+namespace at {
+namespace native{
+
+namespace {
+
+template <template <class...> class set_type, typename scalar_t>
+std::tuple<Tensor, Tensor> _unique_cpu_template(
+    const Tensor& self,
+    const bool return_inverse) {
+  const Tensor& input = self.contiguous();
+  set_type<scalar_t> set(
+      input.data<scalar_t>(), input.data<scalar_t>() + input.numel());
+  Tensor output = input.type().tensor({static_cast<long long>(set.size())});
```

Comment:
I think the cast should be to `int64_t`. It only happens to be an alias for `long long` in your compiler

Reply:
Done